docs/docs/streamr-network/incentives/network-incentives.md

```diff
@@ -30,6 +30,109 @@ The introduction of token economics defines three new roles: [Sponsor](../networ
+- Forced unstaking related
+    - Delegator-initiated after `maxQueueSeconds`, to Operator then to delegator(s) ([source](https://github.com/streamr-dev/network-contracts/blob/master/packages/network-contracts/contracts/OperatorTokenomics/Operator.sol#L441))
+    - Operator-initiated: leave penalty for leaving too early, to protocol ([source](https://github.com/streamr-dev/network-contracts/blob/master/packages/network-contracts/contracts/OperatorTokenomics/Sponsorship.sol#L332))
+- Flagging, voting, and kicking related
+    - stake returned to the Operator that gets kicked out of a Sponsorship (after penalty deducted)
+    - reviewer rewards to Operator: peer-reviewing flags results in a small reward when you vote with the majority
+    - flagger rewards to Operator: flagging a node that then gets kicked out of a Sponsorship results in a small reward
+    - leftover slashings go to protocol: after kicked node gets slashed and rewards paid, there maybe DATA tokens left over that should belong to no one in particular
+    - forfeited locked stake to protocol: raising flags then getting kicked out before they resolve (flag-stake forfeited)
+- Profit sharing related (see [below section](#profit-sharing) for more details)
+    - Protocol fee during profit sharing
+    - "fisherman reward" to Operator: finding other Operators that have too much unwithdrawn earnings results getting a share of the profits ([source](https://github.com/streamr-dev/network-contracts/blob/master/packages/network-contracts/contracts/OperatorTokenomics/Operator.sol#L483))
+
+<!-- above list should be exhaustive in order to be a useful reference! If some other flow/transfer is found, add here! -->
+
+![Sponsor's DATA is sent to stakers via withdraws, earnings in Sponsorship contract become profit in Operator contract, profit is shared between delegators](@site/static/img/DATA-flows-high-level.jpg)
+
+### DATA token flows
+
+There are two processes in the normal operation of Streamr tokenomics: the staking process and the delegation process.
+
+Staking process DATA flows:
+- Stakers send DATA tokens to the Sponsorship contract via [the `stake` method](https://github.com/streamr-dev/network-contracts/blob/master/packages/network-contracts/contracts/OperatorTokenomics/OperatorPolicies/StakeModule.sol#L12)
+- Sponsors send DATA tokens to the Operator contract by calling the `sponsor` method, or `transferAndCall`, or simply transferring (not recommended, the system will lose track of who sponsored)
+- Every second, the remaining sponsorship is allocated to the stakers in proportion to their stake. The allocation is governed by the [StakeWeightedAllocationPolicy](https://github.com/streamr-dev/network-contracts/blob/master/packages/network-contracts/contracts/OperatorTokenomics/SponsorshipPolicies/StakeWeightedAllocationPolicy.sol), as of 2024 the only available allocation policy
```

Comment:
Does this refer to the process where the sponsorship yields earnings to the operators? Maybe could reword this somehow.